.changeset/strong-crabs-flow.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-scaffolder-react': major
```

Comment:
Let's make this minor. You are right that it's technically breaking, but it's in the types and arguably basically a bug fix.

Reply:
👍 I've made the required changes. Also want to make sure regarding the first point in the NOTE to typecast the schema as needed. Review and let me know if changes are required whenever you're not busy :)